core/server/web/site/app.js

```diff
@@ -44,9 +44,14 @@ module.exports = function setupSiteApp() {
     // you can extend Ghost with a custom redirects file
     // see https://github.com/TryGhost/Ghost/issues/7707
     customRedirects.use(siteApp);
+
     // More redirects
     siteApp.use(adminRedirects());
 
+    // Force SSL if required
+    // must happen BEFORE serving assets and BEFORE routing
```

Comment:
Can we go the extra step of adding a why to this comment?

The old comment had no reason (my fault) and now I don't remember why it was the other way around, so maybe let's not make the same mistake again 😂

Reply:
👍